packages/@react-spectrum/s2/src/Picker.tsx

```diff
@@ -101,6 +106,11 @@ interface PickerButtonProps extends PickerStyleProps, ButtonRenderProps {}
 const inputButton = style<PickerButtonProps | AriaSelectRenderProps>({
   ...focusRing(),
   ...fieldInput(),
+  outlineStyle: {
+    default: 'none',
+    isFocusVisible: 'solid',
```

Comment:
did we just forget this earlier?

Reply:
honestly im not too sure why this was taken out before. there was a comment before that said "The focus ring shows up after menu close if default and isFocusVisible from focusRing() aren't added to this definition" but i haven't been able to reproduce unless im not understanding it correctly.